tools/common/Assembly.cs

```diff
@@ -449,8 +449,20 @@ void AddFramework (string file)
 		{
 			if (Driver.GetFrameworks (App).TryGetValue (file, out var framework) && framework.Version > App.SdkVersion)
 				ErrorHelper.Warning (135, "Did not link system framework '{0}' (referenced by assembly '{1}') because it was introduced in {2} {3}, and we're using the {2} {4} SDK.", file, FileName, App.PlatformName, framework.Version, App.SdkVersion);
-			else if (Frameworks.Add (file))
-				Driver.Log (3, "Linking with the framework {0} because it's referenced by a module reference in {1}", file, FileName);
+			else {
+#if MTOUCH
+				var strong = (framework == null) || (App.DeploymentTarget >= (App.IsSimulatorBuild ? framework.VersionAvailableInSimulator ?? framework.Version : framework.Version));
```

Comment:
Should we promote this to a property off assembly?

Reply:
I'm a "one, two, many" guy :)
Also I don't see how it can be turned into a property since it's based on `framework`

Reply:
That's fine, good point.